### ISSUE TITLE
version 0.6.1 of metamask/sdk-react is broken

### ISSUE THREAD

**Issue by equalogix** [MetaMask/metamask-sdk#338]
**Describe the bug**
I updated `sdk-react` to version `0.6.1` and it broke everything. The `useConnect` and `switchOrAddNetwork` custom hooks have been removed from the package, not sure why or whether it was intended, my the code will need to change as a result, but not sure how, as no docs are provided.

**Comment by abretonc7s** [comment]
Hi @equalogix ,

apologies for the issue, we have moved the wagmi hooks into @metamask/sdk-react-ui since they are wrapper around wagmi which added overhead on the mobile connection that we are now trying to optimize.
Please use `@metamask/sdk-react-ui` and instanciate with:
```
import { MetaMaskUIProvider } from '@metamask/sdk-react-ui';

<MetaMaskUIProvider sdkOptions={...} > <Component {...pageProps} /></MetaMaskUIProvider>
```

**Comment by abretonc7s** [comment]
Good point, useSD() wasn't reexported directly from `sdk-react-ui` but you should have access from `@metamask/sdk-react`.
We will export it also from `sdk-react-ui` in next version.
Thank you

**Comment by abretonc7s** [comment]
Did you wrap your code inside the MetaMaskProvider?

As per the example on https://github.com/MetaMask/metamask-sdk/tree/main/packages/examples/create-react-app/src 
```
import React from 'react';
import ReactDOM from 'react-dom/client';
import './index.css';
import App from './App';
import reportWebVitals from './reportWebVitals';
import { MetaMaskProvider } from '@metamask/sdk-react';

const root = ReactDOM.createRoot(
  document.getElementById('root') as HTMLElement
);

root.render(
  <React.StrictMode>
    <MetaMaskProvider debug={false} sdkOptions={{
      logging:{
          developerMode: false,
        },
        communicationServerUrl: process.env.REACT_APP_COMM_SERVER_URL,
        checkInstallationImmediately: false, // This will automatically connect to MetaMask on page load
        dappMetadata: {
          name: "Demo React App",
          url: window.location.host,
        }
    }}>
      <App />
    </MetaMaskProvider>
  </React.StrictMode>
);

// If you want to start measuring performance in your app, pass a function
// to log results (for example: reportWebVitals(console.log))
// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
reportWebVitals();
```

**Comment by abretonc7s** [comment]
There is a new PR to export useSDK from the sdk-react-ui package.
We are trying to publish the patch on npm today.

Thanks for reporting the issue.

**Comment by abretonc7s** [comment]
Fixed on 0.6.2 where sdk-react-ui exports useSDK hook.
